less/index.less

```diff
@@ -48,12 +46,154 @@ table {
 	border-spacing: 0;
 }
 
+//variables
+@import "variables.less";
+@standout: white;
+@font:'Roboto', sans-serif;
+@pfont:'Roboto', sans-serif;
+
+
+//mix-ins
+.bordered() {
+	border-bottom: 1px dashed #00b09b;
+}
+
+.background(){
+	background: linear-gradient(to right, #00b09b, #96c93d);
+}
 /* Set every element's box-sizing to border-box */
 * {
     box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-size: 62.5%;
+	line-height: 15px;
+	font-family: @font;
+}
+h1 { 
+	font-size: 3rem;
+	font-weight: bold;
+	@media @mobile{ 
+		margin-top: 20px;
+	} 
+}
+	
+	h2{ 
+	font-size: 2rem;
+	font-weight: bold;
+}
+	
+	h3 { 
+	font-size: 1.8rem;
+}
+
+
+
+
+header{ 
```

Comment:
This should be nested in "body", and the h1 above should be nested under header.